Improve Aero ROS sample apps

shakthi-prashanth-m opened the pull request:
Improvements to
- Takeoff land
- Telemetry simple
- Flight modes

Commit:
ros-examples: below changes

1. Renamed examples: mavros_* -> aero_*.
2. Added launch files to examples.
3. Using catkin tools instead of catkin_make.
4. Ran clang formatter and other minor changes
5. Applied ROS C++ coding style.
6. Added example description in the source file.

Commit:
ros-examples: below changes

1. Renamed examples: mavros_* -> aero_*.
2. Added launch files to examples.
3. Using catkin tools instead of catkin_make.
4. Ran clang formatter and other minor changes
5. Applied ROS C++ coding style.
6. Added example description in the source file.

Commit:
aero_takeoff_land: Improved output format.

Commit:
aero_takeoff_land: Improved output format.

Commit:
aero_takeoff_land: Refactoring.

1. AeroDrone class abstracts Intel Aero Drone actions arm, takeoff and land.
2. Updated maintainer in package.xml
3. Cmake changes.

Commit:
aero_takeoff_land: Refactoring.

1. AeroDrone class abstracts Intel Aero Drone actions arm, takeoff and land.
2. Updated maintainer in package.xml
3. Cmake changes.

Commit:
aero_takeoff_land: re-organising AeroDrone class.

Commit:
aero_takeoff_land: re-organising AeroDrone class.

Commit:
aero_takeoff_land: Below changes.

1. Added proper debug logs.
2. Added units while printing altitude.

Commit:
aero_takeoff_land: Below changes.

1. Added proper debug logs.
2. Added units while printing altitude.

Commit:
Improvements to aero_takeoff_land, aero_telemetry_simple.

1. Simplified aero_takeoff_land by abstracting AeroDrone actions in its class.
2. Simplified aero_telemetry_simple by abstracting AeroDrone telemetry in its class.

Commit:
Improvements to aero_takeoff_land, aero_telemetry_simple.

1. Simplified aero_takeoff_land by abstracting AeroDrone actions in its class.
2. Simplified aero_telemetry_simple by abstracting AeroDrone telemetry in its class.

Commit:
aero_takeoff_land: Replaced mavros/global_position/global by mavros/home_position/home.

Commit:
aero_takeoff_land: Replaced mavros/global_position/global by mavros/home_position/home.

Commit:
Update aero_telemetry_simple.launch

Commit:
Update aero_telemetry_simple.launch

Commit:
aero_telemetry_simple: Added missing aero_drone_telemetry.cpp

Commit:
aero_telemetry_simple: Add aero_drone_telemetry.cpp.

Commit:
Merge branch 'master' of https://github.com/shakthi-prashanth-m/sample-apps

Commit:
aero_telemetry_simple: Add aero_drone_telemetry.h

Commit:
aero_flight_modes: Improvements.

1. Added AeroDrone class to keep consistent with other examples.

Commit:
aero_flight_modes: Improvements

1. Add AUTHOR info.
2. Correct header file macro.
3. Update comment.

Commit:
aero_flight_modes: Remove extra line in launch file.

Commit:
aero_flight_modes: Correct maintainer email.

anitha-suresh-zz reviewed (commented):
Could you please change commit message header, as discussed.

anitha-suresh-zz commented:
As discussed earlier, code changes look good to me, we could change the commit headers.

Commit:
Improve

1. Version change to 0.1.0.
2. Used pragma once instead of conventional approach.
3. Other minor changes.

Commit:
Improve ROS sample apps

1. Version change to 0.1.0
2. Used pragma once instead of conventional approach
3. Other minor changes

Commit:
Merge branch 'master' of https://github.com/shakthi-prashanth-m/sample-apps
